lib/src/tests/Utils.spec.tsx

```diff
@@ -0,0 +1,9 @@
+import { getSectionName } from ':/utils';
+
+describe('Utils functions must work well to ensure a proper functioning of the library', () => {
+  it("getSectionName must return only section names and undefined if it truly can't", async () => {
+    expect(getSectionName('root_section_widget')).toBe('section');
+    expect(getSectionName('root_section', true)).toBe('section');
+    expect(getSectionName('root_widget')).toBe(undefined);
+  });
```

Comment:
Please in smaller tests help to understand better things.
```suggestion
describe('getSectionName, () => {
  it("should extract section name from widget id", async () => {
    expect(getSectionName('root_section_widget')).toBe('section');
  });
  
  it("should extract section name from section id", async () => {
    expect(getSectionName('root_section', true)).toBe('section');
  });
  
  it("should return undefined when the id is not valid", async () => {
    expect(getSectionName('root_section')).toBeUndefined();
  });
```